components/FlexibleHeader/examples/FlexibleHeaderHorizontalPagingExample.m

```diff
@@ -146,6 +146,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
 
 - (void)commonMDCFlexibleHeaderViewControllerInit {
   self.fhvc = [[MDCFlexibleHeaderViewController alloc] initWithNibName:nil bundle:nil];
+  self.fhvc.headerView.sharedWithManyScrollViews = YES;
```

Comment:
Curiosity: Why does this need to change?  Was the behavior previously broken for this example?